srcpkgs/iputils/template

```diff
@@ -1,39 +1,28 @@
 # Template file for 'iputils'
 pkgname=iputils
-version=20180629
-revision=3
-patch_args="-Np1"
+version=20190515
+revision=1
 wrksrc="${pkgname}-s${version}"
+build_style=meson
+configure_args="-DNO_SETCAP_OR_SUID=true -DUSE_IDN=false
+ -DBUILD_TFTPD=false -DBUILD_NINFOD=false"
+hostmakedepends="pkg-config docbook2x docbook-xsl-ns"
 makedepends="libcap-devel libressl-devel"
 depends="libcap-progs"
 short_desc="Useful utilities for Linux networking (including ping)"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
 license="BSD-3-Clause, GPL-2.0-or-later"
 homepage="https://github.com/iputils/iputils"
-distfiles="https://github.com/${pkgname}/${pkgname}/archive/s${version}.tar.gz"
-checksum=da14105291dd491f28ea91ade854ed10aee8ba019641c80eed233de3908be7c5
+distfiles="https://github.com/iputils/iputils/archive/s${version}.tar.gz"
+checksum=9b5125eb5ef9f4e947ad8fdddcf77f538f53b8f47b53eb5bc5347cb16d01c8fd
 
 alternatives="
  ping:ping:/usr/bin/${pkgname}-ping
  ping:ping.8:/usr/share/man/man8/${pkgname}-ping.8
  ping6:ping6:/usr/bin/${pkgname}-ping
  ping6:ping6.8:/usr/share/man/man8/${pkgname}-ping.8"
 
-do_build() {
-	make CC="$CC" LD="$LD" CCOPTOPT="$CFLAGS" LDFLAGS="$LDFLAGS" \
-		USE_NETTLE=no USE_GNUTLS=no USE_GCRYPT=no USE_IDN=no \
-		${makejobs} \
-		arping clockdiff ping rdisc tracepath
-}
-
-do_install() {
-	vbin ping ${pkgname}-ping
-	for f in clockdiff arping rdisc tracepath; do
-		vbin ${f}
-	done
-	for f in clockdiff arping ping rdisc tracepath; do
-		vman doc/${f}.8
-	done
+post_install() {
```

Comment:
`iputils-ping.8` exists but `iputils-ping` for alternatives does not.

Reply:
fixed